src/main/java/com/lucas/plugintest/BanBlockCommand.java

```diff
@@ -44,9 +41,7 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
             return false;
         }
 
-        String reason = "Sem motivo específicado";
-
-
+        String reason = "Sem motivo específico";
         //se os arguments forem > 1, significa que a motivo, senao ele nao tera e subira a msg padrao da var
         if(args.length > 1){
```

Comment:
Isn't it possibly to get the second argument as a String? Like /ban <player> "<reason>", so it is not needed to make this kind of processing of the arguments, as we can maybe in the future add more args to the command (like time for the ban, or any other parameters)

Reply:
its a good idea, but, for the ban time, i thin that is better to get a new command "tempban"